tds/src/test/java/thredds/server/ncss/controller/grid/GridCoverageSubsettingTest.java

```diff
@@ -193,7 +193,8 @@ public void shouldSubsetGrid() throws Exception {
       System.out.printf("v.getShape()=%s%n", Misc.showInts(v.getShape()));
       assertArrayEquals(expect.shape, v.getShape());
 
-      System.out.printf("ProjectionRect=%s%n", prect.toString2(6));
+      System.out.printf("Expected ProjectionRect=%s%n", expect.rect.toString2(6));
+      System.out.printf("Actual ProjectionRect=%s%n", prect.toString2(6));
       assertTrue(expect.rect.closeEnough(prect));
```

Comment:
Can we not add *more* print statements to the tests? It's 2016, we have loggers. :grin: